tech-committee-charter.md

```diff
@@ -61,6 +61,30 @@ The development process will include a process for the TC to oversee and approve
 
 The TC and entire technical community will follow any processes as may be specified by the Cloud Native Computing Foundation relating to the intake and license compliance review of contributions, including the CNCF IP Policy.
 
+### Sponsorship Requirements
+
+The TC will establish and maintain sponsorship across the SIGs and subprojects of OpenTelemetry.
+
+SIGs will be classified into three classifications of Sponsorship requirement:
```

Comment:
It may be useful to include examples of such existing SIGs in the PR description to make the classification easier to understand.

Reply:
We discussed this in both the GC + TC meetings today.  PTAL at meeting notes and the video recordings.

I agree examples are useful.  I think that's an exercise we'll do over the next week, but I don't think it belongs in the PR.  I'll add the results and examples in the PR description.

Reply:
>I don't think it belongs in the PR. I'll add the results and examples in the PR description.

I agree. That's what I meant: to add to the description, not to `tech-committee-charter.md`.